source/docs/software/examples-tutorials/drivesim-tutorial/drivetrain-model.rst

```diff
@@ -0,0 +1,132 @@
+Step 2: Creating a Drivetrain Model
+===================================
+In order to accurately determine how your physical drivetrain will respond to given motor voltage inputs, an accurate model of your drivetrain must be created. This model is usually created by measuring various physical parameters of your real robot. In WPILib, this drivetrain simulation model is represented by the ``DifferentialDrivetrainSim`` class.
+
+Creating a ``DifferentialDrivetrainSim`` from Physical Measurements
+-------------------------------------------------------------------
+One way to creating a ``DifferentialDrivetrainSim`` instance is by using physical measurements of the drivetrain and robot -- either obtained through CAD software or real-world measurements. This constructor takes the following parameters:
```

Comment:
We should add a note to prefer real-world measurements as teams CADs nearly always differ than whats actually built.